mappings/net/minecraft/util/math/EulerAngle.mapping

```diff
@@ -14,3 +14,6 @@ CLASS net/minecraft/class_2379 net/minecraft/util/math/EulerAngle
 	METHOD method_10256 getPitch ()F
 	METHOD method_10257 getYaw ()F
 	METHOD method_10258 getRoll ()F
+	METHOD method_35845 getWrappedPitch ()F
+	METHOD method_35846 getWrappedYaw ()F
+	METHOD method_35847 getWrappedRoll ()F
```

Comment:
Document the range of the return value. For example, would it be `[-180, 180)` or `[0, 360)`?